site/source/guides/flashing/flashing-firmware.rst

```diff
@@ -3,24 +3,19 @@
 ===================
 Flashing (Firmware)
 ===================
-This page is for existing Librem Mini owners ONLY.  This will add StartOS-specific tweaks to your system, such that it will operate just as a Start9 Pure bought from the `Start9 Store <https://store.start9.com/collections/servers/products/server-pure>`_.
+This page is for Server Pro / Purism Librem Mini users ONLY.  This firmware is custom for these devices.
 
-The source code can be viewed on Purism's `firmware git repo <https://source.puri.sm/firmware/releases/-/tree/master/librem_mini_v2/custom>`_.
+The source code can be viewed on Purism's `firmware git repo <https://source.puri.sm/firmware/releases/-/tree/master/librem_mini_v2/custom>`_.  You will need a USB flash drive, formatted FAT32, to flash the firmware to your server.
 
-#. `Download the firmware from Purism's git repository <https://source.puri.sm/firmware/releases/-/raw/master/librem_mini_v2/custom/pureboot-librem_mini_v2-basic_usb_autoboot_blob_jail-Release-27.1.rom.gz>`_ and extract it.  You'll need a USB flash drive, formatted FAT32.
-    
-    .. warning:: USB drive **must** be formatted as FAT32
-
-#. Copy or move the resulting ``pureboot-librem_mini_v2-basic_usb_autoboot_blob_jail-Release-27.1.rom`` file onto your USB drive, then eject the drive and insert it into your (powered down) Librem Mini.  USB 3.0 (blue ports) are faster.
-
-    .. note:: You'll need a monitor and keyboard plugged into your Librem Mini for this operation
+    .. caution:: USB drive **must** be formatted FAT32
 
-#. Turn on the Librem Mini while pressing the ``ESC`` key on the keyboard repeatedly while you see this screen:
+#. `Download the latest firmware from Purism's git repository <https://source.puri.sm/firmware/releases/-/blob/master/librem_mini_v2/custom/pureboot-librem_mini_v2-basic_usb_autoboot_blob_jail-Release-28.rom.gz>`_ and extract it.
```

Comment:
```suggestion
#. `Download the latest firmware from Purism's git repository <https://source.puri.sm/firmware/releases/-/raw/master/librem_mini_v2/custom/pureboot-librem_mini_v2-basic_usb_autoboot_blob_jail-Release-28.rom.gz>`_ and extract it.
```

Reply:
Direct link to file is better IMO: more direct, less confusing

Reply:
Agreed